packages/nextjs/app/builders/0x5ba397662e0Dc4569199De084C340Cd15b151716/page.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+// import Image from "next/image";
```

Comment:
Is there a reason why you don't want to use a image? 
I think it would be nice to have one.

Reply:
The link from Github was not resolving. May I create a new assets folder and upload directly an image to the public directory or is it best to use another source link for the image?